### PR TITLE
[RW-1760][risk=no] Error handling for cohort search

### PR DESCRIPTION
**Pull request by dolbeew**
Add error handling for for final api calls in cohort search

### CODE REVIEW

File: ui/src/app/cohort-search/search-group-item/search-group-item.component.css [hunk 1]
```diff
@@ -37,3 +37,7 @@
   font-size: 12px;
   color: #333333;
 }
+
+clr-icon[shape^=error-standard] {
```
Comment (freemabd1) [inline ui/src/app/cohort-search/search-group-item/search-group-item.component.css]: is the ^ a typo?

File: ui/src/app/cohort-search/overview/overview.component.ts [hunk 2]
```diff
@@ -24,16 +26,19 @@ export class OverviewComponent {
...
     name: new FormControl('', [Validators.required]),
     description: new FormControl()
   });
 
+  error: boolean;
   stackChart = false;
   showGenderChart = true;
   showComboChart = true;
   showConflictError = false;
+  subscription: Subscription;
```
Comment (freemabd1) [inline ui/src/app/cohort-search/overview/overview.component.ts]: Are we using this subscription anywhere in this component?

File: ui/src/app/cohort-search/overview/overview.component.css [hunk 3]
```diff
@@ -21,6 +21,10 @@
   border-bottom: none;
 }
 
+clr-icon[shape^=error-standard] {
```
Comment (freemabd1) [inline ui/src/app/cohort-search/overview/overview.component.css]: Is ^ a typo?

Comment (dolbeew) [reply ui/src/app/cohort-search/overview/overview.component.css]: I know it looks weird but that's the correct CSS syntax to target a specific attribute and value

File: ui/src/app/cohort-search/overview/overview.component.ts (hunk 2 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/overview/overview.component.ts]: Good catch. I ended up using the async pipe in the template instead. Removed

File: ui/src/app/cohort-search/search-group-item/search-group-item.component.css (hunk 1 above)
Comment (dolbeew) [reply ui/src/app/cohort-search/search-group-item/search-group-item.component.css]: Not a typo, see above comment

**Pull request merged**